packages/common/src/transformUserInfoMessages.test.ts

```diff
@@ -0,0 +1,119 @@
+import { ChannelMessage, DisplayableMessage, InfoMessagesType, MessageType } from '@quiet/types'
+import { transformUserInfoMessages } from './transformUserInfoMessages'
+
+describe('transformUserInfoMessages', () => {
+  const ownerNickname = 'owner'
+  const communityName = 'dev'
+  const userNickname = 'bob'
+
+  it('user joined - message in channel', () => {
+    const message: DisplayableMessage = {
+      id: 'id',
+      type: MessageType.Info,
+      message: InfoMessagesType.USER_JOINED,
+      createdAt: 1,
+      date: '1',
+      nickname: userNickname,
+      isRegistered: false,
+      isDuplicated: false,
+      pubKey: 'pubKey',
+    }
+
+    const transformedMessage = transformUserInfoMessages(ownerNickname, communityName, userNickname, message)
+    const expectedTransformedMessage = {
+      id: 'id',
+      type: 3,
+      message:
+        '**@bob** has joined and will be registered soon. 🎉 [Learn more](https://github.com/TryQuiet/quiet/wiki/Quiet-FAQ)',
```

Comment:
In cases like this we should point to a particular FAQ entry. I should specify this but if I don't, create a placeholder and link to it. The title can be "How does username registration work?"